web/src/components/filings-calendar/tax-access/TaxAccessBody.tsx

```diff
@@ -0,0 +1,26 @@
+import { Heading } from "@/components/njwds-extended/Heading";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { ReactElement } from "react";
+
+interface Props {
+  isStepOne: boolean;
+  showHeader: boolean;
+}
+
+export const TaxAccessBody = (props: Props): ReactElement => {
+  const { Config } = useConfig();
+
+  const getHeader = (): string => {
+    return props.isStepOne ? Config.taxAccess.stepOneHeader : Config.taxAccess.stepTwoHeader;
+  };
+
+  return (
+    <div className="margin-y-3">
+      {props.showHeader && (
+        <Heading level={2} styleVariant="h4" className="margin-bottom-4">
```

Comment:
pebble - aaron mentioned the calendar text is a level 2, given where this is placed, should this be a level 2 as well?